exampleSite/content/search.md

```diff
@@ -0,0 +1,26 @@
++++
```

Comment:
what for this file???

Reply:
This is where search results are rendered.